lib/cloud/transport/common.js

```diff
@@ -86,7 +87,7 @@ export default class CommonTransport {
     if (param.original) {
       originalRecord = new Record(_type, param.original);
     }
-    let record = func(incomingRecord, originalRecord);
+    let record = func(incomingRecord, originalRecord, pool);
```

Comment:
On one hand, this is even better than py-skygear because the pg connection is only created when the connection is needed. But, on the other hand, I do not understand the importance since the user can just import the pool from the `pg` module.